data/sidebar.ts

```diff
@@ -96,7 +96,17 @@ const sidebarContent: SidebarSection[] = [
     slug: "/send-notifications",
     desc: "Learn how to send and debug notifications using Knock.",
     pages: [
-      { slug: "/triggering-workflows", title: "Triggering workflows" },
+      {
+        slug: "/triggering-workflows",
+        title: "Triggering workflows",
+        pages: [
+          { slug: "/overview", title: "Overview" },
+          { slug: "/api", title: "Via the API" },
+          { slug: "/schedules", title: "On a schedule" },
+          { slug: "/events", title: "By an event" },
```

Comment:
```suggestion
          { slug: "/events", title: "From an event" },
```
maybe from or with, I'm thinking of the sentence 'Trigger a workflow [preposition] [article] [noun]'